src/index.html

```diff
@@ -19,9 +19,376 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!-- HEADER -->
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="nav__item"
+            data-qa="nav-hover"
+          >
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- END HEADER -->
+
+    <!-- MAIN -->
+    <main class="cards">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Image card"
+          class="card__img"
+        />
+        <a
+          href="#!"
+          class="card__title"
+        >
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__rating">
+          <div class="stars">
```

Comment:
```suggestion
          <div class="stars stars--4">
```
The number of active stars should depend on the class with the modifier